po/bn.po

```diff
@@ -251,431 +251,431 @@ msgstr "দিন 2: সকাল"
 
 #: src/SUMMARY.md:117
 msgid "Paths"
-msgstr ""
+msgstr "Paths"
 
 #: src/SUMMARY.md:118
 msgid "Filesystem Hierarchy"
-msgstr ""
+msgstr "Filesystem Hierarchy"
 
 #: src/SUMMARY.md:120
 msgid "Luhn Algorithm"
-msgstr ""
+msgstr "Luhn অ্যালগরিদম"
 
 #: src/SUMMARY.md:121
 msgid "Strings and Iterators"
-msgstr ""
+msgstr "Strings and Iterators"
 
 #: src/SUMMARY.md:124
 msgid "Day 3: Morning"
-msgstr ""
+msgstr "তৃতীয় দিনঃ সকাল"
 
 #: src/SUMMARY.md:129
 msgid "Generics"
-msgstr ""
+msgstr "Generics"
 
 #: src/SUMMARY.md:130
 msgid "Generic Data Types"
-msgstr ""
+msgstr "সাধারণ ডেটা ধরণ"
 
 #: src/SUMMARY.md:131
 msgid "Generic Methods"
-msgstr ""
+msgstr "সাধারণ পদ্ধতিগুলো"
 
 #: src/SUMMARY.md:132
 msgid "Monomorphization"
-msgstr ""
+msgstr "Monomorphization"
 
 #: src/SUMMARY.md:133
 msgid "Traits"
-msgstr ""
+msgstr "Traits"
 
 #: src/SUMMARY.md:134
 msgid "Trait Objects"
-msgstr ""
+msgstr "Trait Objects"
 
 #: src/SUMMARY.md:135
 msgid "Deriving Traits"
-msgstr ""
+msgstr "Deriving Traits"
 
 #: src/SUMMARY.md:136
 msgid "Default Methods"
-msgstr ""
+msgstr "ডিফল্ট পদ্ধতি"
 
 #: src/SUMMARY.md:137
 msgid "Trait Bounds"
-msgstr ""
+msgstr "Trait Bounds"
 
 #: src/SUMMARY.md:138
 msgid "impl Trait"
-msgstr ""
+msgstr "impl Trait"
 
 #: src/SUMMARY.md:139
 msgid "Important Traits"
-msgstr ""
+msgstr "গুরুত্বপূর্ণ Traits"
 
 #: src/SUMMARY.md:140
 msgid "Iterator"
-msgstr ""
+msgstr "Iterator"
 
 #: src/SUMMARY.md:141
 msgid "FromIterator"
-msgstr ""
+msgstr "FromIterator"
 
 #: src/SUMMARY.md:142
 msgid "From and Into"
-msgstr ""
+msgstr "From and Into"
 
 #: src/SUMMARY.md:143
 msgid "Read and Write"
-msgstr ""
+msgstr "Read and Write"
 
 #: src/SUMMARY.md:144
 msgid "Drop"
-msgstr ""
+msgstr "Drop"
 
 #: src/SUMMARY.md:145
 msgid "Default"
-msgstr ""
+msgstr "ডিফল্ট"
 
 #: src/SUMMARY.md:146
 msgid "Operators: Add, Mul, ..."
-msgstr ""
+msgstr "অপারেটরঃ Add, Mul, ..."
 
 #: src/SUMMARY.md:147
 msgid "Closures: Fn, FnMut, FnOnce"
-msgstr ""
+msgstr "Closures: Fn, FnMut, FnOnce"
 
 #: src/SUMMARY.md:149
 msgid "A Simple GUI Library"
-msgstr ""
+msgstr "একটি সাধারণ GUI  লাইব্রেরি"
 
 #: src/SUMMARY.md:151
 msgid "Day 3: Afternoon"
-msgstr ""
+msgstr "তৃতীয় দিনঃ বিকাল"
 
 #: src/SUMMARY.md:153
 msgid "Error Handling"
-msgstr ""
+msgstr "ত্রুটি পরিচালনা"
 
 #: src/SUMMARY.md:154
 msgid "Panics"
-msgstr ""
+msgstr "Panics-প্যানিক"
 
 #: src/SUMMARY.md:155
 msgid "Catching Stack Unwinding"
-msgstr ""
+msgstr "Catching Stack Unwinding"
 
 #: src/SUMMARY.md:156
 msgid "Structured Error Handling"
-msgstr ""
+msgstr "স্ট্রাকচার্ড এরর হ্যান্ডলিং"
 
 #: src/SUMMARY.md:157
 msgid "Propagating Errors with ?"
-msgstr ""
+msgstr "কিসের সাথে ত্রুটি প্রচার করা হচ্ছে?"
 
 #: src/SUMMARY.md:158
 msgid "Converting Error Types"
-msgstr ""
+msgstr "ত্রুটি প্রকার রূপান্তর"
 
 #: src/SUMMARY.md:160
 msgid "Deriving Error Enums"
-msgstr ""
+msgstr "Enums ত্রুটি আহরণ"
 
 #: src/SUMMARY.md:161
 msgid "Dynamic Error Types"
-msgstr ""
+msgstr "গতিশীল ত্রুটির ধরন"
```

Comment:
I think গতিশীল is more related to something physical and not appropriate here. ডায়নামিক a isn't bad choice in this context.